pkg/operator/controller/ingress/status.go

```diff
@@ -735,3 +740,37 @@ func computeDNSStatus(ic *operatorv1.IngressController, wildcardRecord *iov1.DNS
 
 	return conditions
 }
+
+// checkZoneInConfig - private utility to check for a zone in the current config
+func checkZoneInConfig(dnsConfig *configv1.DNS, zone configv1.DNSZone) bool {
```

Comment:
Can we create a unit test for this utility function?

Reply:
Yes absolutely will do